config/longrun_configs/amip_n1_shortrun.yml

```diff
@@ -1,24 +1,24 @@
-job_id: "amip_n1_shortrun"
-run_name: "amip_n1_shortrun"
-moist: "equil"
-vert_diff: "true"
-rad: "gray"
-z_elem: 50
-dz_top: 3000
-dz_bottom: 30
-h_elem: 16
-kappa_4: 1e16
-z_stretch: false
-rayleigh_sponge: true
 alpha_rayleigh_uh: 0
 alpha_rayleigh_w: 10
+apply_limiter: false
 dt_cpl: 150
-dt: "150secs"
 dt_rad: "1hours"
+dt_save_to_sol: "100days"
+dt: "150secs"
```

Comment:
`dt` should be above `dt_cpl` for this to be alphabetical. I don't think it really matters as long as we have a convention, but if we want to unify this across repos, we might want to use the same convention that a function like `sort` would output. For example, we could sort all the files in `config/` by executing `find config/ -type f -exec sort -o {} {} \; ` from `ClimaCoupler.jl/`. We might find it useful to use that here and in atmos, land, etc.

Reply:
That's a good point! I guess not everyone is using VSCode, this is more universal. Just changed it.